### PR TITLE
Implement Mortgage Calculator Input Form

### LINKED ISSUE #4: Implement mortgage calculator input form
Create input form that will be used to enter mortgage data as well as personal finance info.
Form must be responsive and include validation.

### PR DESCRIPTION
**Pull request by kandloic**
## Description
Implemented mortgage calculator form.
- Created InputForm component
- Added validation
- Added tooltips to describe each input field

## Related Issue
Fixes #4 

## Changes Made
- [X] Feature added
- [ ] Bug fixed
- [X] Code refactored
- [ ] Documentation updated
- [ ] Other (specify)

## Checklist

- [X] I have tested these changes locally
- [X] The changes do not introduce new warnings or errors
- [X] Documentation has been updated (if applicable)

## Additional Comments
N/A

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (kandloic) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: PR template to use moving forward

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 .vscode/
+.idea/
```
Comment (kandloic) [inline .gitignore:3]: @Nico242001 j'ai remarqué que tu utilises intellij so j'ai ajouté ce `.idea` folder pour que git ne track pas ce folder

File: styles/Home.module.css
Comment (kandloic) [inline styles/Home.module.css:1]: no longer used

File: tailwind.config.js
Comment (kandloic) [inline tailwind.config.js:1]: typo fix

File: vercel.json
Comment (kandloic) [inline vercel.json:1]: no longer needed

**Review by Nico242001 — COMMENTED** [review]
Looks good!

**Pull request merged**